pkg/utils/commands.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import "fmt"
+
+type Commad struct {
+	Name        string
+	Description string
+}
+
+var availableCommands = []Commad{
+	{"digger help", "Display help information"},
+	{"digger version", "Display version information"},
+	{"digger apply", "Apply digger configuration"},
+	{"digger plan", "Plan digger configuration"},
+	{"digger lock", "Lock Terraform state"},
```

Comment:
there is digger unlock too. Also it doesn't lock terraform state, it rather locks terraform projects on pull request level